data/Pandora.Api/V1/Helpers/ObjectDefinition.cs

```diff
@@ -97,6 +98,9 @@ public class ApiObjectDefinition
     [JsonPropertyName("referenceName")]
     public string? ReferenceName { get; set; }
 
+    [JsonPropertyName("referenceIsCommonType")]
+    public bool? ReferenceIsCommonType { get; set; }
```

Comment:
Rather than doing this against each API type, given [v2 of the Data API](https://github.com/hashicorp/pandora/blob/67702bd186a64af9cd6d3975c9a693ce67d8b638/tools/data-api/internal/endpoints/routing.go#L12-L35) exposes per-service level feature-toggles - perhaps we should a) expose and then b) lean on those toggles (e.g. via a new endpoint `/v1/(resource-manager|microsoft-graph)/settings`)?

Per my understanding at least CommonTypes are supposed to enabled/disabled at a Data Source (e.g. Resource Manager/Microsoft Graph) level - rather than toggled on a per-Service/API Version/Resource level?